cargo-scout/src/main.rs

```diff
@@ -15,9 +16,15 @@ use structopt::StructOpt;
 )]
 enum Command {
     Fmt(FmtOptions),
+    Fix(FixCommand),
     Lint(LintOptions),
 }
 
+#[derive(StructOpt)]
+enum FixCommand {
+    Fmt(LintOptions),
```

Comment:
We probably want to use `FmtOptions` here, right?

Reply:
By LintOptions I mean something that returns lints, but I might have picked a poor name here

Reply:
Sorry, I could have explained myself better :)

I meant that right now `cargo-scout fix fmt --help` shows the same options as `cargo-scout lint --help`, which shows options that does not seem to belong to rustfmt, like
```
--no-default-features    Pass the no default features flag to clippy
```

On the other hand, `FmtOptions` seem to have the options related to rustfmt.

Reply:
Oh I m sorry I havent seen your comments!

that makes sense indeed, I'm going to try to find a way to separate options from the logic, which will make it more reliable anyway